addons/pos_paytm/static/src/js/payment_paytm.js

```diff
@@ -0,0 +1,146 @@
+odoo.define('pos_paytm.payment', function (require) {
+"use strict";
+
+const core = require('web.core');
+const rpc = require('web.rpc');
+const PaymentInterface = require('point_of_sale.PaymentInterface');
+const { Gui } = require('point_of_sale.Gui');
+
+const _t = core._t;
+
+let PaymentPaytm = PaymentInterface.extend({
+    init: function (pos, payment_method) {
+        this._super(...arguments);
+    },
+
+    send_payment_request: async function (cid) {
+        /**
+         * Override
+         */
+        this._super.apply(this, arguments);
+        const line = this.pos.get_order().selected_paymentline;
+        const order = this.pos.selectedOrder;
+        let retry = localStorage.getItem(order.uid);
```

Comment:
why use local storage to save the retry number?
a variable in the model would be simpler and sufficient